src/extras/PMREMGenerator.js

```diff
@@ -672,6 +666,10 @@ function _getBlurShader( lodMax, width, height ) {
 
 		name: 'SphericalGaussianBlur',
 
+		extensions: {
```

Comment:
Adding the `extensions` object like that is actually not supported since all other flags of `extensions` become undefined. The idea is to use:
```js
shaderMaterial.extensions.shaderTextureLOD = true;

Reply:
ah, good call, thanks!